activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -97,7 +97,10 @@ class Numeric
   #  1234567.to_s(:human, precision: 1,
   #                   separator: ',',
   #                   significant: false)                   # => "1,2 Million"
-  def to_formatted_s(format = :default, options = {})
+  def to_s(*args)
```

Comment:
Unfortunately no if we want to support [Float#to_s](http://ruby-doc.org/core-2.2.2/Float.html#method-i-to_s).